v19.2/movr.md

```diff
@@ -0,0 +1,85 @@
+---
+title: MovR
+summary: The MovR application uses CockroachDB to store information about vehicles, users, and rides.
+toc: true
+---
+
+MovR is a fictional vehicle-sharing company created to demonstrate CockroachDB's features.
+
+## Overview
+
+The MovR example consists of three components:
+
+- The `movr` database, which contains several tables that hold vehicle, user, and ride data. The `movr` database is built into `cockroach demo` and `cockroach workload`.
+- The `movr` dataset, which consists of rows of data that populate tables in the `movr` database. The `movr` dataset is built into `cockroach demo` and `cockroach workload`.
+- The MovR application is a fully-functional vehicle-sharing application, written in Python. All of MovR application source code is open-source, and available on the [movr](https://github.com/cockroachdb/movr) GitHub repository.
```

Comment:
All of the mentions (or at least the first mention) of `cockroach demo` and `cockroach workload` could link to the docs for those commands.

Reply:
Done!